app/src/main/java/me/ccrama/redditslide/Activities/MainActivity.java

```diff
@@ -510,8 +510,8 @@ public boolean onCreateOptionsMenu(Menu menu) {
             } else {
                 inflater.inflate(R.menu.menu_subreddit_overview, menu);
             }
-            //Only show the "Share Slide" menu item if the user doesn't have Pro installed
-            if (SettingValues.tabletUI) {
+            // Hide the "Share Slide" menu if the user has Pro installed
```

Comment:
I also changed this comment to indicate what the next block of code is about to do, not what it will not do if it fails.